app/assets/stylesheets/dashboard.scss

```diff
@@ -313,6 +313,20 @@ p.debate-status {
       height: 60px;
     }
 
+    .left .image {
```

Comment:
Nazwa klasy kontenera `.left` jest niesemantyczna. Co jak kiedyś po jakimś redesignie ten element wyląduje po prawej stronie? Nazwa powinna bardziej odzwierciedlać to co się znajduje wewnątrz kontenera, a nie gdzie on powinien być. 

Wiem, że to Ty nie dodałeś, ale zawsze dobrze pomalutku takie rzeczy poprawiać :)